### DISCUSSION TITLE
Can't use the Workspaces, no tutorial

### DISCUSSION THREAD

**Discussion by friedafeuerstein** [Zettlr/Zettlr#4565] [Q&A]
Hey there, I just installed Zettlr and realized that it was not the right version for my Mac (M1). Then deleted and uninstalled everything and wanted to install the Apple Silicone version. Now I tried this a few times and it always happens the same: I can install the app, but the tutorial and the first files are missing. I also can't create a new workspace myself. I don't know what I'm doing wrong. The other Mac version also has this error. 
Also, I was wondering where exactly in the settings I can specify where the files and directories are stored in Zettlr by default. Sure, exporting works, but where can I set where Zettlr puts its file directory? 
Thanks!

**Comment by nathanlesage** [comment]
Hey! First of all, I'm sorry that it didn't work out from the start of your experience with the app. I assume you downloaded the 2.3.0 version, right? If so, I would like to invite you to wait until September 1st, which is when the new version 3.0.0 will be released that will improve a ton of things. If you want to, you can already switch to the most recent beta release, that you can find here, on this page under the "releases" tab.

To get the tutorial back, there are two ways. The easy one for Mac is to open a Finder window, click "Go", hold down ALT and click on Library. Then, navigate to Application Support -> Zettlr. If you didn't yet do anything in Zettlr, you can just move the entire folder to the Trash, because there won't be any statistics etc recorded yet. Then, launch the (correct) app. It should see that there is no config, and therefore mount the tutorial again. If you switch to 3.0 beta (or wait and use the stable release directly), you should also navigate to your Documents folder and remove the folder Zettlr Tutorial. The new one will be updated and cleaned up.

If you need anything else, I warmly invite you to join the discord (invite link is on our website, in the footer), there is a large and very helpful community that can help you out with whatever you desire!
